Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1258,6 +1267,9 @@
 /* Action sheet item */
 "SHOW_SAFETY_NUMBER_ACTION" = "Show new safety number";
 
+/* notification action */
+"SHOW_THREAD_BUTTON_TITLE" = "Show Thread";
```

Comment:
I suggest we use "thread" only in the code, and exclusively use "conversation" in our copy.